tests/app/main/test_frameworks.py

```diff
@@ -1554,48 +1558,37 @@ def test_make_or_edit_supplier_declaration_shows_correct_page(
 
         self.data_api_client.get_framework.return_value = self.framework(status='open')
         self.data_api_client.get_supplier_framework_info.return_value = self.supplier_framework(
-            **supplier_framework_kwargs
+            application_company_details_confirmed=True,
+            **supplier_framework_kwargs,
         )
 
         response = self.client.get('/suppliers/frameworks/g-cloud-7')
         document = html.fromstring(response.get_data(as_text=True))
 
         assert (
-            document.xpath("//a[normalize-space(string())=$link_label]/@href", link_label=link_label)[0]
+            document.xpath("//a[contains(normalize-space(string()), $link_label)]/@href", link_label=link_label)[0]
```

Comment:
What does adding `contains` give you?

Reply:
The labels now start with a tasklist item number, which I'm not really interested in asserting are a specific value here.